CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 Change Log
 ==========
 
+Unreleased
+----------
+
+**Added**
+
+* Add method :meth:`~.Subreddits.premium` to reflect the naming change in
```

Comment:
```suggestion
* Method :meth:`~.Subreddits.premium` to reflect the naming change in
```
It’s under the `Added` section so I don’t think it’s necessary to include.

Reply:
With this change, this is no longer a complete sentence. Also, there is previous change log entries that are like this.

Reply:
I see, let me rephrase it.

Reply:
Never mind, there seems to be no way to change the current sentence without a full rewrite